### ISSUE TITLE
IBMDBPY on Python Flask app in IBM Cloud not working

### ISSUE THREAD

**Issue by daniielamir** [ibmdbanalytics/ibmdbpy#28]
Hi! I am trying to establish a connection to DB2 thought this package in my python flask application. At the moment and to make it simple my code looks like this: 
`idadb = IdaDataBase(dsn=jdbc)`

I have the correct packages to the requirements txt file as well s import them in the welcome.py file. When I push my code to IBM Cloud I get an error which seems to be because jaydebeapi need Server JRE to run. However the error looks like this: 

idadb = IdaDataBase(dsn=jdbc) #Establish a connection to our DB2-service
1/20/2018 12:05:45 PM   ERR undefined   File "/home/vcap/deps/0/python./lib/python2.7/site-packages/ibmdbpy/base.py", line 282, in __init__
1/20/2018 12:05:45 PM   ERR undefined jpype.startJVM(jpype.getDefaultJVMPath(), '-Djava.class.path=%s' % jarpath)
1/20/2018 12:05:45 PM   ERR undefined   File "/home/vcap/deps/0/python/lib/python2.7/site-packages/jpype/_core.py", line 114, in get_default_jvm_path
1/20/2018 12:05:45 PM   ERR undefined   return finder.get_jvm_path()
1/20/2018 12:05:45 PM   ERR undefined   File "/home/vcap/deps/0/python/lib/python2.7/site-packages/jpype/_jvmfinder.py", line 121, in get_jvm_path
1/20/2018 12:05:45 PM   ERR undefined   jvm = method()
1/20/2018 12:05:45 PM   ERR undefined   File "/home/vcap/deps/0/python/lib/python2.7/site-packages/jpype/_jvmfinder.py", line 164, in _get_from_known_locations
1/20/2018 12:05:45 PM   ERR undefined   for home in self.find_possible_homes(self._locations):
1/20/2018 12:05:45 PM   ERR undefined   File "/home/vcap/deps/0/python/lib/python2.7/site-packages/jpype/_jvmfinder.py", line 95, in find_possible_homes
1/20/2018 12:05:45 PM   ERR undefined   for childname in sorted(os.listdir(parent)):
1/20/2018 12:05:45 PM   ERR undefined   OSError: [Errno 2] No such file or directory: '/usr/lib/jvm'
1/20/2018 12:05:46 PM   OUT undefined   Exit status 1

How can I solve this?

**Comment by rafrodriguez** [comment]
It seems like JVM (Java virtual machine) is not installed in your system, or it might be that it is installed but its environment variables are not properly set, because jpype is failing to find it in /usr/lib/jvm

**Comment by daniielamir** [comment]
Yes, I know. But how can I install it? I know how to install it on my computer. But how can I do it in a python flask application I am pushing to Bluemix? @rafrodriguez

**Comment by dawangandypeng** [comment]
I have the same question actually. I was wondering whether it's possible to configure ODBC on Bluemix?

**Comment by WarioDoor** [comment]
was somebody able to connect in bluemix?
